modules/administration-guide/pages/repairing-the-devworkspace-operator.adoc

```diff
@@ -0,0 +1,89 @@
+:_content-type: PROCEDURE
+:navtitle: Repairing the {devworkspace} Operator
+:description: Repairing the {devworkspace} Operator
+:keywords: devworkspace, operator, repair
+:page-aliases:
+
+[id="repairing-the-devworkspace-operator_{context}"]
+= Repairing the {devworkspace} Operator
+
+.Prerequisites
+
+* An active `{orch-cli}` session as a cluster administrator to the destination {orch-name} cluster. See {orch-cli-link}.
+
+.Procedure
+
+. Delete the `devworkspace-controller` namespace that contains the failing pod.
+
+. Update `DevWorkspace` and `DevWorkspaceTemplate` Custom Resource Definitions (CRD) by setting the conversion strategy to `None` and removing the next several lines of the removed strategy:
++
+[source,yaml]
+----
+spec:
+  ...
+  conversion:
+    strategy: None # note .spec.conversion.webhook is removed here
+status:
+...
+----
++
+TIP: You can find and edit the `DevWorkspace` and `DevWorkspaceTemplate` CRDs in the *Administrator* perspective of the OpenShift web console by searching for `DevWorkspace` in menu:Administration[CustomResourceDefinitions].
++
+NOTE: The `DevWorkspaceOperatorConfig` and `DevWorkspaceRouting` CRDs  have the conversion strategy set to `None` by default.
+
+. Remove the {devworkspace} Operator subscription:
++
+----
+$ oc delete sub devworkspace-operator
+----
+
+. Get the {devworkspace} Operator CSVs:
++
+----
+$ oc get csv | grep devworkspace
+----
++
+pass:[<!-- vale CheDocs.Attributes = NO -->]
+
+.Output displayed in the __<devworkspace-operator.vX.Y.Z>__ format
+====
+----
+devworkspace-operator.v0.15.1    DevWorkspace Operator 0.15.1    devworkspace-operator.v0.15.0    Replacing
+
+devworkspace-operator.v0.15.2    DevWorkspace Operator 0.15.2    devworkspace-operator.v0.15.1    Pending
+----
+====
```

Comment:
Avoid using hard-coded version numbers, to avoid unnecessary maintenance overhead.

Use attributes for version numbers.

Even better, consider removing this example, as it is not necessary for understanding the procedure.

Reply:
This is just an example so it should be ok

Reply:
I managed to incorporate the format into the sentence of the stop, and so I removed the example anyway.